lib/re_styled_ppx.re

```diff
@@ -6,8 +6,35 @@ open Parsetree;
 
 let expr = (mapper, expression) =>
   switch (expression.pexp_desc) {
-  | Pexp_extension(({txt: "styled", loc: _}, _payload)) =>
-    Ast_helper.Exp.constant(Pconst_integer("42", None))
+  | Pexp_extension(({txt: "styled", loc}, _payload)) =>
+    switch (expression.pexp_desc) {
+      | Pexp_constant(Pconst_string(str, delim)) =>
+        let loc_start =
+          switch (delim) {
+          | None => expression.pexp_loc.Location.loc_start
+          | Some(s) => {
+              ...expression.pexp_loc.Location.loc_start,
+              Lexing.pos_cnum:
+                expression.pexp_loc.Location.loc_start.Lexing.pos_cnum
+                + String.length(s)
+                + 1,
+            }
+          };
+        let container_lnum = loc_start.Lexing.pos_lnum;
+        let ast =
+          Css_lexer.parse_string(
+            ~container_lnum,
+            ~pos=loc_start,
+            str,
+            Css_parser.declaration_list,
+          );
+
+          Console.log(ast);
+        Ast_helper.Exp.constant(Pconst_integer("42", None))
```

Comment:
It feels like this is almost at a point where this line could be replaced with `Css_to_emotion.render_stylesheet(ast)` and it would work?